src/app/components/Transactions/ConsensusTransactions.tsx

```diff
@@ -94,19 +91,7 @@ export const ConsensusTransactions: FC<ConsensusTransactionsProps> = ({
                     pr: 3,
                   }}
                 >
-                  {!!ownAddress && transaction.sender === ownAddress ? (
```

Comment:
Is this intentional / any idea how to re-implement marking the current address?

https://93f0c866.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qp533xjqg7rjn9xglk6fgp3pykdscsk6vs3m2tuz

| Before | After |
| --- | --- |
|   ![explorer dev oasis io_mainnet_consensus_address_oasis1qp533xjqg7rjn9xglk6fgp3pykdscsk6vs3m2tuz](https://github.com/oasisprotocol/explorer/assets/3758846/1047c6ba-558f-470b-8185-809c9bf72501)     |    ![93f0c866 oasis-explorer pages dev_mainnet_consensus_address_oasis1qp533xjqg7rjn9xglk6fgp3pykdscsk6vs3m2tuz](https://github.com/oasisprotocol/explorer/assets/3758846/308c2e8b-e052-46f5-9449-6028f6528635)    |

Reply:
Not intentional on my part. I have a vague memory seeing this chance coming in from somewhere else, I being surprised, but I can't recall the specifics. I'll just re-introduce this now.

Reply:
Restored in https://74d1e081.oasis-explorer.pages.dev/mainnet/consensus/address/oasis1qp533xjqg7rjn9xglk6fgp3pykdscsk6vs3m2tuz

Reply:
Only fixed consensus

Reply:
Oops. Now fixed on paratimes, too.